certif/app/controllers/authenticated/sessions/import.js

```diff
@@ -122,4 +117,16 @@ export default class ImportController extends Controller {
   reset() {
     this.isImportStepOne = true;
   }
+
+  _handleApiError(responseError) {
+    const error = responseError?.errors[0];
+    switch (error.code) {
+      case 'CSV_HEADERS_NOT_VALID':
+        return `pages.sessions.import.step-one.errors.${error.code}`;
+      case 'CSV_DATA_REQUIRED':
+        return `pages.sessions.import.step-one.errors.${error.code}`;
+      default:
+        return ENV.APP.API_ERROR_MESSAGES.INTERNAL_SERVER_ERROR.I18N_KEY;
+    }
+  }
```

Comment:
```suggestion
  _handleApiError(responseError) {
    const errorCodes = ['CSV_HEADERS_NOT_VALID', 'CSV_DATA_REQUIRED'];
    const error = responseError?.errors[0];
    return errorCodes.includes(error.code) ? 
        `pages.sessions.import.step-one.errors.${error.code}` :
         ENV.APP.API_ERROR_MESSAGES.INTERNAL_SERVER_ERROR.I18N_KEY;
  }
```

Suggestion: De façon à ne pas renvoyer toujours la même ligne de traduction quelque soit le code d'erreur reçu.

Reply:
Le switch est source d'erreur, comme le montre l'exemple d'@alexandre, il ne semble pas apporter de clarté ici

Reply:
j'avais en tête de remplir davantage le switch, avec notamment un traitement sur les 400, 401 etc... Mais ok pour la modif :)